### PR TITLE
Portfolio: Setup

### PR DESCRIPTION
**Pull request by peterdtitan**
Started working on Portfolio Website Project. Made additions and changes to project structure. The following files were added and/or removed: 

- index.html was created and necessary header and body content were added.
- style.css was created and linked to index.html.
- page media assets folder was created.
- correct configuration of the project structure (adding linters) was set up.

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @peterdtitan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] doesn't have a descriptive title and short summary that tells in more detail what you've done in PR
- [ ] doesn't have linters set up correctly please try to fix that  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Peter Okorafor - Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="topnav">
+            <a href="#home" class="logo">Peter Okorafor</a>
+        </div>
```
Comment (Kakalanp) [inline index.html]: Remember to add a `<nav>` element for your hamburger button, that way, navigation will be much easier for the user :wink: 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </header>
+
+    <div class="container">
```
Comment (Kakalanp) [inline index.html]: Remember it is considered good practice to use semantic tags, to follow best practices you should separate sections with `<section>` tags :fire:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+</head>
+<body>
+    <header>
+        <div class="topnav">
+            <a href="#home" class="logo">Peter Okorafor</a>
+        </div>
+    </header>
+
+    <div class="container">        
+        <p class="container headline">Hey There.<br>I'm Peter</p>
```
Comment (Kakalanp) [inline index.html]: Kindly change your title tag to `<h1>`, the main title of a page should always use `<h1>` to follow good practices :100:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        <p class="container job-title">I am a Software Developer</p>
```
Comment (Kakalanp) [inline index.html]: Kindly change your subtitle tags to `<h2>`, the subtitles of a page should always use `<h2>`,`<h3>`,`<h4>`,`<h5>` in a hierarchical way to follow good practices :sunglasses:

File: index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        <p class="container job-description">
+                I can help you build a product, feature or website.
+                Look through some of my work and experience!
+                If you like what you see and have a project you need coded, do not hestiate to contact me.
+        </p>
+    </div>
+
+
+
+    <div class="links">
```
Comment (Kakalanp) [inline index.html]: Don't forget that it's also considered good practice to use `<ul>` tags for lists, the links and `<a>` tags are fine, but you could try to contain them in different `<li>`'s to make sure your project looks professional :briefcase:

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  height: 312.03px;
+  left: 620.1px;
+  top: 317px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.topnav {
```
Comment (Kakalanp) [inline css/style.css:25]: [OPTIONAL]

You could also select the HTML elements from the `<header>` tag, that way you wouldn't need to use a new div and class :thinking:

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# Hello there @peterdtitan! hi

### Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

so close

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Good implementation from the Figma design :monocle_face: 

## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (peterdtitan) [reply index.html]: Great tip! Thanks.

File: index.html (hunk 3 above)
Comment (peterdtitan) [reply index.html]: Should have seen that too 😄

File: index.html (hunk 2 above)
Comment (peterdtitan) [reply index.html]: Okay. Change made!  👍🏽

File: index.html [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+    </section>
+
+    <section class="main-content">
```
Comment (Kossi-stack) [inline index.html:43]: - [x] kindly align your content well according to figma design.
- Figma Design
Captureinli
- Your Design
Captureinl

File: README.md [hunk 8]
```diff
@@ -1 +1,51 @@
-# portfolio-build
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Portfolio Page
+
+> This is the mobile version of a personal portfolio page that shows off my accomplishments as a software developer.
+
+![screenshot](./assets/screenshot.png)
+
```
Comment (Kossi-stack) [inline README.md:8]: - [ ] [OPTIONAL] Kindly remove empty lines for a clean code

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @peterdtitan ,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. 
 - [ ] Please make sure to add .gitignore file to the praject also don't forget to ignore node_modules.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 7 above)
Comment (peterdtitan) [reply index.html:43]: For the links to external portfolios, isn't it appropriate to only add links I have a portfolio on (github, linkedin and twitter)?

**Comment by peterdtitan** [comment]
Updated content alignment to match Figma designs. The following changes were made: 

- Updated the css styling to align contents correctly as in given design.
- Added Medium and AngelList icons/links.
- Updated README file and cleaned up empty lines.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @peterdtitan, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

## Optional: It is highly suggested to change your PR title from `Feature` to `Portfolio: Setup`. I tried Contacting you on slack but you have not responded. As this was the only thing that's the reason I approved but make sure to change your PR title.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**